setup.cfg

```diff
@@ -57,3 +57,80 @@ per-file-ignores =
 exclude=
     .eggs
     doc
+
+[mypy]
+exclude = properties|asv_bench|doc
+files = .
+show_error_codes = True
+
+# Most of the numerical computing stack doesn't have type annotations yet.
+[mypy-affine.*]
```

Comment:
I have this stuff in `pyproject.toml` Is there another one that needs to be added to that list?

Reply:
hmm, I mostly mindlessly copy/pasted from xarray assuming mypy wasn't setup here. 
I need to do a proper comparison then.